apps/storybook/stories/components/accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,117 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { Accordion as KvibAccordion, AccordionItem, AccordionButton, AccordionPanel } from "@kvib/react/src";
+
+const meta: Meta<typeof KvibAccordion> = {
+  title: "Komponenter/Accordion ",
+  component: KvibAccordion,
+  parameters: {
+    docs: {
+      story: { inline: true },
+      canvas: { sourceState: "shown" },
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof KvibAccordion>;
+
+export const Accordion: Story = {
+  argTypes: {
+    allowMultiple: {
+      description: "If true, multiple items can be expanded at once.",
+      table: {
+        type: { summary: "boolean" },
+        defaultValue: { summary: false },
+      },
+      control: "boolean",
+    },
+    allowToggle: {
+      description: "If true, expanded items may be collapsed again.",
+      table: {
+        type: { summary: "boolean" },
+        defaultValue: { summary: false },
+      },
+      control: "boolean",
+    },
+    defaultIndex: {
+      description: "Sets the initial index for the expanded accordion item.",
+      table: {
+        type: { summary: "string" },
+      },
+      control: "text",
+    },
+    index: {
+      description: "Sets the index for the expanded accordion item.",
+      table: {
+        type: { summary: "string" },
+      },
+      control: "text",
+    },
+    onChange: {
+      description: "Callback for when the accordion expands/collapses.",
+      table: {
+        type: { summary: "function" },
+      },
+      control: "boolean",
+    },
+    reduceMotion: {
+      description: "If true, disables height animation and transition.",
+      table: {
+        type: { summary: Boolean },
+        defaultValue: { summary: false },
+      },
+      control: "boolean",
+    },
+  },
+  args: { allowMultiple: false, allowToggle: false },
```

Comment:
Tror du kan sette "onChange: undefined" inn i args for å fjerne onChange😊

Reply:
Jeg prøvde det forrige uke, men det viste seg at grunnen til at ingenting virket for meg, var at jeg skrev det inn på feil story 😅